### PR TITLE
Minor typos

### PR DESCRIPTION
**Pull request by JakobMiksch**

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -149,10 +149,10 @@ back at vehicle end.
 Use `skills` to describe a problem where not all tasks can be served
 by all vehicles. Job skills are mandatory, i.e. a job can only be
 served by a vehicle that has **all** its required skills. In other
-words: job `j` is eligible to vehicle `v` iff `j.skills` is included
```
Comment (jcoupey) [inline docs/API.md:152]: This is no a typo, but a feature. ;-) Intended to stand for "if and only if", https://en.wikipedia.org/wiki/If_and_only_if

File: docs/API.md [hunk 2]
```diff
@@ -149,10 +149,10 @@ back at vehicle end.
...
+words: job `j` is eligible to vehicle `v` if `j.skills` is included
 in `v.skills`.
 
-In order to ease modeling problems with no skills required, it is
+In order to ease modelling problems with no skills required, it is
```
Comment (jcoupey) [inline docs/API.md:155]: Sounds like yet another US vs rest of the world difference: https://grammarist.com/spelling/model/. Maybe we should simply go with "describing"?

File: docs/API.md [hunk 3]
```diff
@@ -195,7 +195,7 @@ The computed solution is written as `json` on standard output or a file
 | Key         | Description |
 | ----------- | ----------- |
 | `code` | status code |
-| `error` | error message (present iff `code` is different from `0`) |
```
Comment (jcoupey) [inline docs/API.md:198]: Same as above.

**Review by jcoupey — COMMENTED** [review]
Thanks for taking the time to submit a PR.

File: docs/API.md [hunk 4]
```diff
@@ -149,10 +149,10 @@ back at vehicle end.
 Use `skills` to describe a problem where not all tasks can be served
 by all vehicles. Job skills are mandatory, i.e. a job can only be
 served by a vehicle that has **all** its required skills. In other
-words: job `j` is eligible to vehicle `v` iff `j.skills` is included
+words: job `j` is eligible to vehicle `v` if `j.skills` is included
```
[the remark this answers is not in the document]
Comment (jcoupey) [reply docs/API.md:152]: No problem to reformulate if it's more clear.

**Comment by jcoupey** [comment]
@JakobMiksch sorry for the review race condition.

File: docs/API.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (JakobMiksch) [reply docs/API.md:152]: good to know, I did not know about "iff" before

**Pull request closed without merging**